site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md

```diff
@@ -0,0 +1,582 @@
+
+Are you interested in bringing your database game to the next level by unleashing the power of Snowflake’s many DevOps capabilities, empowering you to build safe and robust data pipelines? Well then, this Quickstart is for you!
+
+This quickstart will cover a lot of ground. By the end of this guide, you will have built a robust CI/CD pipeline that automates the deployment of changes to a data pipeline from Git, promotes the separation of dev and prod databases and facilitates monitoring. You'll also learn how to use Snowflake's marketplace to enrich your data without incurring storage costs, get to know the new developer CLI tool and the Visual Studio Code extension!
+
+![DevOps Overview](assets/devops-overview.png)
+
+The data pipeline used as an example in this quickstart makes use of data available on the Snowflake marketplace to come up with the perfect spot for a family vacation. Let's get started!
+
+> aside negative
+> Note: The Git integration, Create or Alter and Jinja templating are still in preview.
+
+### Prerequisites
+
+- Familiarity with Snowflake
+- Fundamentals of data engineering. You can use [Intro to Data Engineering with Snowpark Python](https://quickstarts.snowflake.com/guide/data_engineering_with_snowpark_python_intro/index.html) to refresh your memory
+- Familiarity with Git repositories and GitHub
+
+### What you will learn
+
+You will learn about the following Snowflake features in this Quickstart:
+
+- Native Git Integration
+- Declarative Database Change Management
+- Parametrization of SQL scripts with Jinja
+- GitHub Actions (CI/CD) integration
+- Snowflake’s Visual Studio Code extension
+- SnowflakeCLI tool
+- Data Sharing/Marketplace (instead of ETL)
+- LLM interference with Snowflake Cortex
+
+### What you will need
+
+You will need the following things before starting:
+
+- Snowflake
+  - **A [Snowflake Account](https://signup.snowflake.com/)**
+  - **A Snowflake user with ACCOUNTADMIN permissions**
+  - **Anaconda Terms & Conditions accepted**. See [Using Third-Party Packages](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-packages#getting-started).
+- GitHub
+  - A GitHub account. If you don't already have one, you can [create one for free](https://github.com/).
+
+### What you will build
+
+During this Quickstart you will accomplish the following things:
+
+- Establish a connection between Snowflake and a GitHub repository
+- Setup access to Snowflake Marketplace data
+- Create a vectorized Python UDF making use of dynamic file access
+- Create a data engineering pipeline to process data from multiple sources
+- Orchestrate the pipeline with tasks
+- Make declarative changes to the pipeline with Create or Alter
```

Comment:
wondering if we should make it more clear that it is not "either create or alternatively alter", but rather a combined new feature. Suggestion: Create-or-Alter. If you change this there are many places in the doc that need to be updated consistently.